ipi/engine/motion/driven_dynamics.py

```diff
@@ -120,6 +129,27 @@ def EDAforces(self, time):
         forces = Constants.e * Z @ E  # array of shape (nbeads,3xNatoms)
         return forces
 
+    def dipole_dtime(self, bead=-1):
+        """Compute the time derivative of the electric dipole of the system"""
```

Comment:
maybe explain that you compute by the chain rule. what is the purpose of adding this function, just out of curiosity? could also be added to description